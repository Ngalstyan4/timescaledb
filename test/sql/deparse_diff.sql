SET client_min_messages = 'fatal';
\set ECHO none
\o /dev/null

-- turn off alignment to avoid + signs at the end of lines
\a
-- turn off headers
\t
\ir include/deparse_create.sql
\a
\t

\o :TEST_OUTPUT_DIR/results/deparse_create.out
\d+ (public|"customSchema").*

\a
\t
\o /dev/null
\ir include/deparse_recreate.sql
\a
\t

\o :TEST_OUTPUT_DIR/results/deparse_recreate.out
\d+ (public|"customSchema").*

\o
\set ECHO 'all'
\! diff -y --suppress-common-lines  -W 240 ${TEST_OUTPUT_DIR}/results/deparse_create.out ${TEST_OUTPUT_DIR}/results/deparse_recreate.out