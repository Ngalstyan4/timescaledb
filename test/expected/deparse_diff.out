SET client_min_messages = 'fatal';
\set ECHO none
\! diff -y --suppress-common-lines  -W 240 ${TEST_OUTPUT_DIR}/results/deparse_create.out ${TEST_OUTPUT_DIR}/results/deparse_recreate.out
